.changeset/healthy-timers-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': patch
+---
+
+Resolving issue #4921
```

Comment:
I'm not sure if this requires a changeset, but if it does, this phrasing doesn't tell anything. Would you mind updating the changeset with a concrete explanation of what's being solved?

Reply:
We can disable the check if needed.

Please refer to [the GUIDELINES](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/7bd2b2aaf68c21277097166a9a51eb72ae239b34/GUIDELINES.md) before we review further.

> All changes to the core codebase (excluding tests, auxiliary scripts, etc.) must be documented in a changelog, except for purely cosmetic or documentation changes.

Also review our [CHANGELOG](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/6ba452dea4258afe77726293435f10baf2bed265/CHANGELOG.md) for a reference of how to write a proper changeset.

Reply:
Is it important to add a changeset for a small issue like this?

Reply:
I think this PR should not have a changelog entry.

Reply:
> I think this PR should not have a changelog entry.

Pretty much agree.